mysite/search/migrations/0014_populate_icons_for_existing_projects_LATER.py

```diff
@@ -29,11 +29,6 @@ def forwards(self, orm):
         # Populate icons for all existing projects.
         for orm_project in orm['search.project'].objects.all():
             model_project = Project.objects.get(pk=orm_project.pk)
-            print "Populating icon for", model_project
-            model_project.populate_icon_from_ohloh()
-            # We know this is unsafe.
-            # If this fails, it might be because
-            # populate_icon_from_ohloh doesn't exist anymore.
```

Comment:
@onceuponatimeforever , on #1524, you can see why I changed the migrations in django, those are custom migrations and needed to be changed after I had removed ``populate_icon_from_ohloh()``  else would have caused  ``AttributeError`` .